packages/cli/src/events/WorkflowStatistics.ts

```diff
@@ -22,14 +22,7 @@ export async function workflowExecutionCompleted(
 	}
 
 	// Get the workflow id
-	let workflowId: number;
-	try {
-		workflowId = parseInt(workflowData.id as string, 10);
-		if (isNaN(workflowId)) throw new Error('not a number');
-	} catch (error) {
-		LoggerProxy.error(`Error "${error as string}" when casting workflow ID to a number`);
-		return;
-	}
+	const workflowId = workflowData.id!;
```

Comment:
No idea about the context here but I'd validate rather than assert.

Reply:
added a simple return. I don't think this is supposed to be `undefined` ever, and it's likely just a case of a "wrong" type.